.cirrus.yml

```diff
@@ -4,9 +4,11 @@ web_shard_template: &WEB_SHARD_TEMPLATE
   only_if: "changesInclude('.cirrus.yml', 'DEPS', 'lib/web_ui/**', 'web_sdk/**') || $CIRRUS_PR == ''"
   environment:
     # As of March 2020, the Web shards needed 16G of RAM and 4 CPUs to run all framework tests with goldens without flaking.
+    WEB_SHARD_COUNT: 12
     CPU: 4
     MEMORY: 16G
-    WEB_SHARD_COUNT: 4
+    CHROME_NO_SANDBOX: true
+    GOLD_SERVICE_ACCOUNT: ENCRYPTED[3afeea5ac7201151c3d0dc9648862f0462b5e4f55dc600ca8b692319622f7c3eda3d577b1b16cc2ef0311b7314c1c095]
```

Comment:
@Piinks Do you know what GOLD_SERVICE_ACCOUNT is used for, this was added to web shards for flutter/flutter but was missing on the engine side, so I'm also adding this value here.

Reply:
If the tests are passing without it, it's probably not needed here.

Reply:
Yeah, seems like they failed. so I'll try another run without it.

Reply:
What are we missing to migrate this tests to LUCI?